application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/wolframalpha/misunderstoodqueries/Tips.java

```diff
@@ -0,0 +1,42 @@
+package org.togetherjava.tjbot.commands.mathcommands.wolframalpha.misunderstoodqueries;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonRootName;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@SuppressWarnings("unused")
+@JsonRootName("tips")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Tips {
+    @JacksonXmlProperty(isAttribute = true)
+    int count;
+    List<Tip> tips;
```

Comment:
private

Reply:
this is not how to collect the tips. u need jackson annotations for unwrapping. see the other examples.
also, the variable name has to be `tip` (or better, use the annotation to change it)